### ISSUE TITLE
Chapter 7: FILES command line error

### ISSUE THREAD

**Issue by Xylopyrographer** [PacktPublishing/Mastering-Embedded-Linux-Programming-Third-Edition#74]
Howdy. Another error. MELP, 3rd Ed.

Page 219, at step 5 this line:

`FILES_${PN} += "/usr/share/*"`

is in error.

Should be:

`FILES:${PN} += "/usr/share/*"`

or to be more consistent (as per the Yocto manual):

` FILES:${PN} += "${datadir}"`

The `devtool build bubblewrap` command will fail otherwise.

Took me a bit to sort that one out.

**Comment by fvasquez** [comment]
`FILES_${PN}` uses the old BitBake override syntax which was valid at the time the book was written.

https://wiki.koansoftware.com/index.php/Override_syntax_change_from_Yocto_honister_3.4

`FILES:${PN}` should work with the Yocto 3.1 (Dunfell) LTS release which is required for Chapter 7. Let me know if it doesn't.

**Comment by Xylopyrographer** [comment]
Hi Frank: Good information, thank you. Yes, the `FILES:${PN}` syntax is what I used.

When starting the book I tried to match versions of each package with that used in the text but wasn't able to figure out how to active that across the board, especially with kernel versions and the relocation of some sources. Hence I restarted from the beginning, picking where I could packages with the lowest version. For Yocto, I did pick a newer version as that seemed to address number of bugs.
